src/main/java/pl/edu/mimuw/chess/pieces/BishopPiece.java

```diff
@@ -0,0 +1,39 @@
+package pl.edu.mimuw.chess.pieces;
+
+import pl.edu.mimuw.chess.ChessBoard;
+import pl.edu.mimuw.chess.ChessColor;
+import pl.edu.mimuw.chess.V2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static pl.edu.mimuw.chess.V2.*;
+
+public class BishopPiece extends AbstractPiece {
+
+  private static final List<List<V2>> POSSIBLE_MOVES = generatePossibleMoves();
+
+  public BishopPiece(V2 position, ChessColor color) {
+    super(position, color, "♗", "♝");
+  }
+
+  private static List<List<V2>> generatePossibleMoves() {
```

Comment:
sortujemy metody - statyczne na koniec, a w ogólności to pierw `public`, potem `private`